thermoconScreens/thermocon.ui

```diff
@@ -810,7 +810,7 @@
          <enum>QFrame::Sunken</enum>
         </property>
         <property name="text">
-         <string>XXXXXXXXX</string>
+         <string>ca://${DEV}:PB_RANGE_RBV</string>
```

Comment:
Seems like pydm + designer decided to make its own additional adjustments here

I think maybe it just changed the default value (pre-connect) value of a qlabel text field at some point